packages/styles/fonts.css

```diff
@@ -10,46 +10,110 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-:root,
-:host {
-    font-family: var(--spectrum-global-font-family-base);
-    font-size: var(--spectrum-alias-font-size-default);
+/* stylelint-disable */
 
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+:root {
```

Comment:
We may need to look at how we're using this... `:root` won't work in a element, IIRC, which means it'll need to be updated to `:root, :host` to work at that level.

Reply:
@adixon-adobe This may be the only note that _must_ be solved before we move forward, but hopefully you have a minute to get into this soon as I'd love to have this updated.

Reply:
Agreed -- I missed this in the conversion. I'm gonna write up a spectrum-css issue and just hard-code this final fix.

Reply:
We process this out in other contexts. `.spectrum` -> `:root, :host`...we should probably add that so we can keep up-to-date. It would be great if we could get Spectrum CSS to update, but I'd not bet very much money on it...

Reply:
oh, and question, is this a bug in spectrum-css, or something we should be converting?

Reply:
They don't really trade in `:host` as it's a Shadow DOM selector, so our conversion...until we convince them to only make a web component distribution of their code 🤔 😜

Reply:
thanks -- wasn't clear where things were at there.